docs/examples/ratify-verify-azure-cmd.md

```diff
@@ -1,296 +1,361 @@
-                        "version": "1.0",
-                        "trustPolicies": [
-                            {
-                            "name": "default",
-                            "registryScopes": [
-                                "*"
-                            ],
-                            "signatureVerification": {
-                                "level": "strict"
-                            },
-                            "trustStores": [
-                                "ca:certs"
-                            ],
-                            "trustedIdentities": [
-                                "*"
-                            ]
-                        }]
-                    }
-                },
-                {
-                    "name":"sbom",
-                    "artifactTypes" : "org.example.sbom.v0",
-                    "nestedReferences": "application/vnd.cncf.notary.signature"
-                }
-            ]
-        }
-    }
-    EOF
+    notation sign --signature-format cose --key $KEY_NAME $IMAGE
     ```
 
-3. Discover the signatures
+## View the signed images associated with signatures
 
-    ```bash
-    # Discover full graph of supply chain content
-    ratify discover -s $IMAGE_DIGEST_REF
-    ```
+Signed images can be viewed with the `notation list` command
 
-4. Verify the full graph of supply chain content
+```bash
+notation list $IMAGE
+```
+## Create and configure Azure Workload Identity
 
-    ```bash
-    # Verify full graph
-    ratify verify -s $IMAGE_DIGEST_REF
-    ```
+Ratify pulls artifacts from a private Azure Container Registry using Workload Federated Identity in an Azure Kubernetes Service cluster. For an overview on how workload identity operates in Azure, refer to the [documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation). You can use workload identity federation to configure an Azure AD app registration or user-assgined managed identity. The following workflow include Workload Identity configuration.
+
+### Set up Workload Identity with AKS, ACR and AKV:
+The official steps for setting up Workload Identity on AKS can be found [here](https://azure.github.io/azure-workload-identity/docs/quick-start.html).
+
+> Note: if you have identity authenticaton issue in your local machie, you can switch to use Azure Cloud Shell the complete this section.
+
+1. Configure environment variables.
+
+```bash
+export IDENTITY_NAME=<Identity Name>
+export GROUP_NAME=<Azure Resource Group Name>
+export SUBSCRIPTION_ID=<Azure Subscription ID>
+export TENANT_ID=<Azure Tenant ID>
+export ACR_NAME=<Azure Container Registry Name>
+export AKS_NAME=<Azure Kubernetes Service Name>
+export KEYVAULT_NAME=<Azure Key Vault Name>
+export RATIFY_NAMESPACE=<Namespace where Ratify deployed, defaults to "gatekeeper-system">
+export CERT_DIR=<Directory storing TLS certs>
+export NOTARY_PERM_NAME=<Name of cerficicate file uploaded to Key Vault>
```

Comment:
```suggestion
export NOTARY_PEM_NAME=<Name of cerficicate file uploaded to Key Vault>
```

Reply:
also you can get rid of the `CERT_DIR` since we are relying on the chart auto generating this now.

Reply:
That's cool!